org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -2703,6 +2777,175 @@ class MarkedString {
 	}
 }
 
+interface WorkDoneProgressNotification {
+	def WorkDoneProgressKind getKind()
+}
+ 
+
+@JsonRpcData
+class WorkDoneProgressBegin implements WorkDoneProgressNotification {
```

Comment:
Please add `Since` information as JavaDoc similar to how it was done for other types.